lib/gui/app/models/flash-state.ts

```diff
@@ -45,10 +45,10 @@ export function isFlashing(): boolean {
  * The flag is used to signify that we're going to
  * start a flash process.
  */
-export function setFlashingFlag() {
+export async function setFlashingFlag() {
 	// see https://github.com/balenablocks/balena-electron-env/blob/4fce9c461f294d4a768db8f247eea6f75d7b08b0/README.md#remote-methods
 	try {
-		electron.ipcRenderer.invoke('disable-screensaver');
+		await electron.ipcRenderer.send('disable-screensaver');
```

Comment:
Docs don't mention the return type of `send` but I suspect it doesn't return a promise like `invoke`, so there's no need to `await` anything.

Reply:
indeed it doesn't care there's no listener